tests/Python/test_python.py

```diff
@@ -0,0 +1,35 @@
+from buddy_mlir.ir import *
```

Comment:
Please use Lit and FileCheck to test the code.

Reply:
When I run `ninja check-buddy` ，I got an error like this.
```shell
check:37'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
            4: ModuleNotFoundError: No module named 'buddy_mlir' 
```
I think this is because we don't export the right python path. The CMakeLists.txt tells us the `BUDDY_MLIR_PYTHON_PACKAGES_DIR` is `${BUDDY_BUILD_DIR}/python_packages` 
https://github.com/buddy-compiler/buddy-mlir/blob/ce466bac3af5401341c29ffc34b11f235e58546d/CMakeLists.txt#L89
And `config.buddy_python_packages_dir ` equals to `BUDDY_MLIR_PYTHON_PACKAGES_DIR`.
https://github.com/buddy-compiler/buddy-mlir/blob/ce466bac3af5401341c29ffc34b11f235e58546d/tests/lit.site.cfg.py.in#L37-L39
However, I set generated python packages to `${BUDDY_BUILD_DIR}/midend/python_packages/mlir_core`. 

And when I add this path to config.buddy_mlir_enable_python_packages, it works. So we need only one path finally.
https://github.com/buddy-compiler/buddy-mlir/blob/ce466bac3af5401341c29ffc34b11f235e58546d/tests/lit.cfg.py#L91-L105